include/aws/common/private/lookup3.c

```diff
@@ -506,12 +516,16 @@ static void hashlittle2(
      * "k[2]&0xffffff" actually reads beyond the end of the string, but
      * then masks off the part it's not allowed to read.  Because the
      * string is aligned, the masked-off tail is in the same word as the
-     * rest of the string.  Every machine with memory protection I've seen
-     * does it on word boundaries, so is OK with this.  But VALGRIND will
-     * still catch it and complain.  The masking trick does make the hash
-     * noticably faster for short strings (like English words).
+     * rest of the string. Every machine with memory protection I've seen
+     * does it on word boundaries, so is OK with this. But VALGRIND and CBMC
+     * will still catch it and complain. CBMC will ignore this type of error
+     * in the code block between the pragmas "CPROVER check push" and
+     * "CPROVER check pop". The masking trick does make the hash noticably
+     * faster for short strings (like English words).
      */
 #ifndef VALGRIND
+#pragma CPROVER check push
+#pragma CPROVER check disable "pointer"
```

Comment:
guard with `#ifdef CBMC` please

Reply:
Done! You must merge first https://github.com/awslabs/aws-c-common/pull/364